### ISSUE TITLE
Please release jackson-module-jaxb-annotations 2.13.4

### ISSUE THREAD

**Issue by onigoetz** [FasterXML/jackson-modules-base#180]
I get the following error from the `maven-enforcer-plugin`:

```
[INFO] --- maven-enforcer-plugin
[WARNING] 
Dependency convergence error for jakarta.activation:jakarta.activation-api:1.2.2 paths to dependency are:
+-my-internal-library:1.0.0-SNAPSHOT
  +-org.glassfish.jersey.media:jersey-media-json-jackson:2.36
    +-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.13.3
      +-jakarta.xml.bind:jakarta.xml.bind-api:2.3.3
        +-jakarta.activation:jakarta.activation-api:1.2.2
and
+-my-internal-library:1.0.0-SNAPSHOT
  +-org.glassfish.jersey.media:jersey-media-json-jackson:2.36
    +-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.13.3
      +-jakarta.activation:jakarta.activation-api:1.2.1
```

Which means that `jackson-module-jaxb-annotations` proposes both 1.2.1 and 1.2.2 of `activation-api` 
A commit was made in november to fix this : https://github.com/FasterXML/jackson-modules-base/commit/199ae53414ea5d44a8143a691c901acab6310756 but was never released, can you please release it ?

**Comment by cowtowncoder** [comment]
Yes, eventually 2.13.4 will be released. Unfortunately the full patch release takes multiple hours so I will wait for a set of fixes; and 2.13 branch is become quite stable now:

https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13.4

so I don't think I will release a full set until July or August.

Theoretically I could release a micro-patch (2.13.3.1) of just `jackson-module-jaxb-annotations` as that takes much less time (since only a single module is released, not all 20+ Jackson components across a dozen repos).

But before doing that, quick question: have you verified that the issue is resolved by using a local snapshot build (or one from Sonatype OSS repo)?

**Comment by onigoetz** [comment]
Fair question, I'll try that and report back

**Comment by onigoetz** [comment]
So I have good and bad news.

Bad news, the commit I mentioned isn't included in the 2.13 branch, but once the change in version is done, it works fine.

Is it possible to cherry-pick https://github.com/FasterXML/jackson-modules-base/commit/199ae53414ea5d44a8143a691c901acab6310756 to the 2.13 branch  and release that as 2.13.3.1 only for this module ?

**Comment by cowtowncoder** [comment]
Ah, it was for 2.14 only. Hmmh. Do you think this is safe change? If so, yes, PR with cherry-pick (or just manually re-creating) is something I'd accept, for 2.13.3.1 micro-patch.

**Comment by onigoetz** [comment]
According to the changelog it's a safe change : https://github.com/eclipse-ee4j/jaf/releases/tag/1.2.2

**Comment by cowtowncoder** [comment]
2.13.4 was released a while ago, closing.

**Comment by K0K0V0K** [comment]
Hey guys!

Maybe i am wrong but i think i can observe this issue in the 2.13.4 version as well.

```
Dependency convergence error for jakarta.activation:jakarta.activation-api:1.2.2 paths to dependency are:
+-somelib:1.0-SNAPSHOT
  +-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.13.4
    +-jakarta.xml.bind:jakarta.xml.bind-api:2.3.3
      +-jakarta.activation:jakarta.activation-api:1.2.2
and
+-somelib:1.0-SNAPSHOT
  +-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.13.4
    +-jakarta.activation:jakarta.activation-api:1.2.1
 ```
 
 I am the only one?

**Comment by cowtowncoder** [comment]
If there are remaining problems, please file a new issue with ref to this one & then someone can propose a PR. I am not sure why first fix did not resolve the problem but typically do not re-open issues marked as resolved since that complicates tracking of issues vs versions, in release notes.
